### PR TITLE
Combined "win" directories, some code-style

### PR DESCRIPTION
**Pull request by jonano614**
- version T13.895 is merged
- win32 and win64 directories are combined
- x86 compilation is fixed
- project filters are added
- code style and some comments in files address.c and block.c

### CODE REVIEW

File: client/address.c [hunk 1]
```diff
@@ -7,7 +7,7 @@
 static const uint8_t bits2mime[64] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
 static uint8_t mime2bits[256];
 
-//intialization of addresses module
+/* intializates the addresses module */
```
Comment (trueserve) [inline client/address.c]: // initializes the address module

**Review by trueserve — CHANGES_REQUESTED** [review]
C++ style comments are OK, they need not be all C style.

**Comment by trueserve** [comment]
This is a bit of a mess with back and forth; can you separate commits based on the final work rather than the work in progress?

I'll do this one for now but please keep this in mind for future work.

**Review by trueserve — APPROVED** [review]

File: client/address.c (hunk 1 above)
Comment (jonano614) [reply client/address.c]: Sorry, typo. I wrote "initization" first, but I decided to change it to "initializes" later.

**Pull request merged**